src/_cffi_src/openssl/src/osrandom_engine.h

```diff
@@ -3,4 +3,7 @@
 #else
 #include <fcntl.h>
 #include <unistd.h>
+#ifdef HAVE_SYS_STAT_H
```

Comment:
This was pulled from the Python patch, but under what circumstances would this not be true but fstat, et al would still be available?